src/registrar/models/domain_request.py

```diff
@@ -1040,17 +1065,12 @@ def reject(self):
         """Reject an domain request that has been submitted.
 
         As side effects this will delete the domain and domain_information
-        (will cascade), and send an email notification."""
+        (will cascade), and send an email notification using send_custom_status_update_email.
```

Comment:
Match the commnent for action_needed, it's much clearer on where the email is being sent out from